roadrunner.rb

```diff
@@ -0,0 +1,131 @@
+# frozen_string_literal: true
+
+# Namespace for all RoadRunner code
+module RoadRunner
+  def self.initialize_report_data
```

Comment:
This module just has class methods far as I could see - could do something like:
```
module RoadRunner
  class << self
  
  def initialize_report_data
    ...
  end
   
  ...  
end
```

Instead of having to do `self.some_method` every time